manual/Website1/package.json

```diff
@@ -6,9 +6,10 @@
     "url": "https://github.com/faceyspacey/remixx.git"
   },
   "author": "Zack Jackson <zack@ScriptedAlchemy.com> (https://github.com/ScriptedAlchemy)",
+  "license": "GPL-3.0-only",
   "scripts": {
-    "manual:prod": "cross-env NODE_ENV=production node node_modules/webpack-dev-server/bin/webpack-dev-server.js",
-    "manual:dev": "cross-env NODE_ENV=development node node_modules/webpack-dev-server/bin/webpack-dev-server.js"
+    "manual:prod": "cross-env NODE_ENV=production webpack-dev-server",
```

Comment:
Revert this, it allows me to pass `node --inspect nodemodules/` and debug builds via chrome console